environment.sh

```diff
@@ -55,18 +54,20 @@ if [[ $exit_code == '0' ]]; then
   echo "Git branch '$branch_name' exists in the remote repository, pulling latest changes..."
   git pull origin $branch_name
 fi
+requirements_modification_time="$(date -r $install_file '+%Y-%m-%d %H:%M:%S')"
 
 # Check if environment exists already
 create_env=$(conda info --envs | grep $env_name | head -n 1)
 if [[ $create_env == '' ]]; then
   # No environment exists with this name
+  echo "Environment $env_name does not exist."
   create_env="yes"
 else
   create_env="no"
 fi
 
+# Check if existing environment needs to be recreated
```

Comment:
This can be simplified:
```bash
if [[ $create_env == '' ]]; then
  echo "Environment $env_name does not exist."
  create_env="yes"
else
  echo "Existing environment found for $env_name."
  one_week_ago=$(date -d "7 days ago" '+%Y-%m-%d %H:%M:%S')
  ...
fi
```